views/default/resources/account/change_password.php

```diff
@@ -32,7 +32,6 @@
 $content = elgg_view_form('user/changepassword', array('class' => 'elgg-form-account'), $params);
 
 if (elgg_get_config('walled_garden')) {
-	elgg_load_css('elgg.walled_garden');
 	$body = elgg_view_layout('walled_garden', array('content' => $content));
```

Comment:
I think we should just pass the title into the layout and get rid of walled garden module wrapper.